packages/slate-editor/src/lib/fetchImageWithProgress/fetchImageWithProgress.ts

```diff
@@ -3,7 +3,7 @@ import { ProgressPromise } from '@prezly/progress-promise';
 import { fetchImageWithReadableStream } from './fetchImageWithReadableStream';
 import { fetchImageWithXmlHttpRequest } from './fetchImageWithXmlHttpRequest';
 
-const CORS_ENABLED_ORIGINS = ['https://cdn.uc.assets.prezly.com'];
+const CORS_ENABLED_ORIGINS = ['https://cdn.uc.assets.prezly.com', 'https://avatars-cdn.prezly.com'];
```

Comment:
I am not sure if this is going to work for oembed screenshots, but don't have a way to test it locally for obvious reasons. Do you think this will work since the screenshots are loaded via restpack.io (the request to avatars-cdn is forwarded)? 🤔

Reply:
> Do you think this will work since the screenshots are loaded via restpack.io (the request to avatars-cdn is forwarded)?

Unlikely. The avatars endpoint _redirects_ the request to the real image location, not forwards.

Reply:
Hmm 🤔